Assets/ConsentManagementProvider/Plugins/iOS/Source/SwiftBridge.swift

```diff
@@ -0,0 +1,334 @@
+    var callbackOnConsentUIReady: СallbackCharMessage? = nil
+    var callbackOnConsentAction: СallbackCharMessage? = nil
+    var callbackOnConsentUIFinished: СallbackCharMessage? = nil
+    var callbackOnErrorCallback: СallbackCharMessage? = nil
+    var callbackOnSPFinished: СallbackCharMessage? = nil
+    var callbackOnSPUIFinished: СallbackCharMessage? = nil
+    var callbackOnCustomConsent: СallbackCharMessage? = nil
+
+// MARK: - Bridge config
+    @objc public func addTargetingParam(campaignType: Int, key: String, value: String){
+        switch CAMPAIGN_TYPE(rawValue: campaignType) {
+        case .GDPR: gdprTargetingParams[key]=value
+
+        case .IOS14: break
+
+        case .CCPA: ccpaTargetingParams[key]=value
+
+        case .none:
+            print("Incorrect campaignType on addTargetingParam")
+        }
+    }
+    
+    @objc public func configLib(
+        accountId: Int,
+        propertyId: Int,
+        propertyName: String,
+        gdpr: Bool,
+        ccpa: Bool,
+        language: SPMessageLanguage,
+        gdprPmId: String,
+        ccpaPmId: String) {
+            self.config = { Config(
+                accountId: accountId,
+                propertyId: propertyId,
+                propertyName: propertyName,
+                gdpr: gdpr,
+                ccpa: ccpa,
+                att: true,
+                language: language,
+                gdprPmId: gdprPmId,
+                ccpaPmId: ccpaPmId
+            )}()
+        }
+    
+    @objc public func addCustomVendor(vendor: String) {
+        config.vendors.append(vendor)
+    }
+
+    @objc public func addCustomCategory(category: String) {
+        config.categories.append(category)
+    }
+
+    @objc public func addCustomLegIntCategory(legIntCategory: String) {
+        config.legIntCategories.append(legIntCategory)
+    }
+    
+    @objc public func clearCustomArrays() {
+        config.vendors = []
+        config.categories = []
+        config.legIntCategories = []
+    }
+    
+    @objc public func dispose() {
+        callbackDefault = nil
+        callbackOnConsentReady = nil
+        callbackOnConsentUIReady = nil
+        callbackOnConsentAction = nil
+        callbackOnConsentUIFinished = nil
+        callbackOnErrorCallback = nil
+        callbackOnSPFinished = nil
+        callbackOnSPUIFinished = nil
+        callbackOnCustomConsent = nil
+    }
+ 
+// MARK: - Manage lib
+    @objc public func loadMessage(authId: String? = nil) {
+        print("PURE SWIFT loadMessage")
+        consentManager.loadMessage(forAuthId: authId)
+    }
+
+    @objc public func onClearConsentTap() {
+        SPConsentManager.clearAllData()
+        myVendorAccepted = .Unknown
+    }
+    
+    @objc public func onGDPRPrivacyManagerTap() {
+        consentManager.loadGDPRPrivacyManager(withId: config.gdprPmId!)
+    }
+    
+    @objc public func onCCPAPrivacyManagerTap() {
+        consentManager.loadCCPAPrivacyManager(withId: config.ccpaPmId!)
```

Comment:
We should avoid the force unwrapping (`.ccpaPmId!`). Instead we could:
```swift
if let pmId = config.ccpaPmId {
  consentManager.loadCCPAPrivacyManager(withId: config.ccpaPmId!)
} else {
  print("Tried to load CCPA pm without ccpa pm id")
}
```

Same for GDPR above.